### PR TITLE
Fix usage of symbol

### PR DESCRIPTION
**Pull request by iwamatsu0430**
#517 
Fix symbol `'xxx` to `Symbol("xxx")`.

### CODE REVIEW

File: airframe-config/src/test/scala/wvlet/airframe/config/ConfigTest.scala [hunk 1]
```diff
@@ -265,7 +265,7 @@ class ConfigTest extends AirframeSpec {
           unused = Some(p)
         })
 
-      unused shouldBe 'defined
+      unused shouldBe Symbol("defined")
```
Comment (xerial) [inline airframe-config/src/test/scala/wvlet/airframe/config/ConfigTest.scala]: Change this to ScalaTest's `shouldBe defined` http://www.scalatest.org/user_guide/using_matchers#workingWithOptions

File: airframe-log/jvm/src/test/scala/wvlet/log/io/IOUtilTest.scala [hunk 2]
```diff
@@ -30,11 +30,11 @@ class IOUtilTest extends Spec {
 
     "find a file" in {
       val buildSbt = IOUtil.findPath("build.sbt")
-      buildSbt shouldBe 'defined
+      buildSbt shouldBe Symbol("defined")
```
Comment (xerial) [inline airframe-log/jvm/src/test/scala/wvlet/log/io/IOUtilTest.scala]: ditto

File: airframe-log/jvm/src/test/scala/wvlet/log/io/IOUtilTest.scala [hunk 3]
```diff
@@ -30,11 +30,11 @@ class IOUtilTest extends Spec {
...
       buildSbt.get.getPath shouldBe "build.sbt"
 
       val notFound = IOUtil.findPath("non-existing-file-path.xxxxxxx")
-      notFound shouldBe 'empty
+      notFound shouldBe Symbol("empty")
```
Comment (xerial) [inline airframe-log/jvm/src/test/scala/wvlet/log/io/IOUtilTest.scala]: ditto

**Comment by iwamatsu0430** [comment]
@xerial 
Thanks for your review.
Could you please check my code again?

**Comment by iwamatsu0430** [comment]
Sorry, CI build failed.
but it look like another matter with my change.
https://travis-ci.org/wvlet/airframe/jobs/552287075#L741-L776

What should I do?

**Comment by xerial** [comment]
The error is unrelated to this PR. Thanks for your contribution!

**Pull request merged**